### ISSUE TITLE
GloDroid v0.7.4 on PinePhone v1.2: Both F-Droid and Aurora Store are unable to install some apps due to "incompatibility"

### ISSUE THREAD

**Issue by volkertb** [GloDroid/glodroid_manifest#200]
Running GloDroid v0.7.4 on PinePhone v1.2 (2GB RAM, UBports Community Edition), I've noticed that F-Droid lists some of the apps in its store (such as [GPS Cockpit](https://f-droid.org/en/packages/org.woheller69.gpscockpit/)) as "Incompatible". Likewise, Aurora Store is filtering some apps that it deems incompatible with this platform from the search results, notably Spotify. When I use a mobile web browser to go to [Spotify's Android download page](https://www.spotify.com/us/download/android/) and tap on the Download button and then tell the browser to open the app store link with the Aurora Store app, I get the option to install the app, but when I tap on "Install", it fails with the messages "Download Failed" and "App not supported". See the screenshots included below.

Neither F-Droid nor Aurora Store provide any clarification as to why these apps are incompatible with this combination of OS and device.

Could it be possible that GloDroid on Aarch64 devices currently lacks backward compatibility with 32-bit ARMv7 apps?

![signal-2022-04-24-145352_001](https://user-images.githubusercontent.com/518023/164977755-37855fff-f9ca-42f0-8d12-fcef38349411.png)

![signal-2022-04-24-145433_001](https://user-images.githubusercontent.com/518023/164977765-2f79a33d-a53e-4783-9de2-61dce535a8bb.png)

**Comment by cristiioan** [comment]
Not sure about APPs on Fdroid, but Spotify has probably some issue with DRM and this is why it is not supported

**Comment by herbsmn** [comment]
I just downloaded GPS Cockpit 2.0 via F-Droid on my 2GB RAM PinePhone running GloDroid v0.8.1 without issue. I don't think GPS currently works, so the app doesn't do much, but it installed without issue.

I haven't come across any F-Droid apps yet which say they are incompatible.

Also, I just downloaded Spotify with the Aurora Store app and it installed without any issues. I haven't tried logging in, but it installed just fine.
